test-cases/manifest.ttl

```diff
@@ -0,0 +1,1335 @@
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix rml: <http://w3id.org/rml/> .
+@prefix rmltest: <http://w3id.org/rml/test/> .
+@prefix test: <http://www.w3.org/2006/03/test-description#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<http://w3id.org/rml/io/test/RMLSTC0001a> a test:TestCase;
+  dcterms:identifier "RMLSTC0001a";
+  rml:input <http://w3id.org/rml/io/test/RMLSTC0001a/input/Friends.json/>;
```

Comment:
namespace should probably `rmltest` for input and output as well.

Reply:
Fixed.